src/pages/LoginPage/LoginPage.jsx

```diff
@@ -1,11 +1,15 @@
 import axios from 'axios';
 import React, { useState } from 'react';
-import { Link, useNavigate as navigate } from 'react-router-dom';
-import "./LoginPage.css"
+import { Link, useNavigate } from 'react-router-dom';
+import "./LoginPage.css";
+import { useAuth } from '../../context/AuthContext';
 
 function LoginPage() {
+    const {login ,setlogin } = useAuth();
     const [email,setEmail] = useState();
     const [password,setPassword] = useState();
+    const navigate = useNavigate();
+
     const onSubmittFunc = async () =>{
```

Comment:
You can rename this somthing like `onSubmitHandler` instead of `onSubmitFunc`

Reply:
done